packages/mobile/src/identity/reducer.ts

```diff
@@ -200,7 +200,7 @@ export const reducer = (
     case Actions.COMPLETE_ATTESTATION_CODE:
       return {
         ...state,
-        ...completeCodeReducer(state, state.numCompleteAttestations + 1),
+        numCompleteAttestations: state.numCompleteAttestations + 1,
```

Comment:
Why is this change needed?

Reply:
this is needed because `completeCodeReducer` should be used only for the case when we set completed codes initially. We can not use it here, because on this line https://github.com/celo-org/celo-monorepo/blob/77df351699e03bd9df96d6ab13eaa3d04a9c1886/packages/mobile/src/identity/reducer.ts#L370 we would assign to `attestationCodes[i]` undefined yet `acceptedAttestationCodes[i]`, which will be defined after this reducer is finishes.